templates/dalite.site

```diff
@@ -0,0 +1,58 @@
+server {
+    listen 80 default_server;
+    server_name _;
```

Comment:
This is nonintuitive, but this is how nginx works. I wanted to filter-out invalid domains on nginx level, whiich should be as easy as speciyfing `server_name daltie.harvardx....` . To cut long story short I need to provide a "default catch-all server", and if I don't first found server will serve that role. 

So this is a default server registered for a nonexistent `server_name` (this is `_`) that returns 404 for invalid domains. If you will go to test dalite: http://149.202.190.106/ you will get 404.